.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,19 @@
+name: Pull request
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  tests:
+    name: Test
+    uses: swiftlang/github-workflows/.github/workflows/swift_package_test.yml@main
+    with:
```

Comment:
Please set `linux_os_versions` to the full list of distributions; the coverage is rather important for this project as it has some distro-specific behaviors.